Source/Atomic/Core/Context.h

```diff
@@ -228,6 +228,27 @@ class ATOMIC_API Context : public RefCounted
     void AddGlobalEventListener(GlobalEventListener* listener) { globalEventListeners_.Push(listener); }
     void RemoveGlobalEventListener(GlobalEventListener* listener) { globalEventListeners_.Erase(globalEventListeners_.Find(listener)); }
 
+    WeakPtr<Engine> engine_;

```

Comment:
Alrighty, on it

![](http://i1.kym-cdn.com/photos/images/newsfeed/000/544/022/2f0.gif)